modules/ROOT/pages/setup.adoc

```diff
@@ -30,10 +30,32 @@ For more information, refer to xref:setup-payload-storage-API.adoc[Set up the Pa
 
 * Mule runtime engine (Mule) 4.2.2 or later (for On-Premises only).
 
-* The Partner Manager Administrator role for all users who need access to B2B message flows in their environment (Production or  Sandbox).
-
 * Keep the default deployment of message flows to CloudHub (for information on CloudHub settings, refer to xref:cloudhub-deploy-options.adoc[CloudHub Deployment Options], or deploy to your own hosted on-premise Mule runtime, by clicking *Stop deploying to CloudHub* and going to  xref:deploy-onpremise.adoc[Deploy On-Premises for Partner Manager]).
 
+* The Partner Manager Administrator role that assigns permission levels for all users who need access to B2B message flows in their environment (Production or  Sandbox).
+
+=== Parter Manager Roles
```

Comment:
Is this text about naming and describing "roles" or "levels of access"?
Most of the following items do not look like role names, but look more like levels that should not be capitalized as if they were names.
If these are labels in the syntax, then they should be enclosed in ` `.